pontoon/contributors/utils.py

```diff
@@ -478,39 +488,70 @@ def get_contribution_timeline_data(user, contribution_type=None, day=None):
         contributions_qs = contributions_map[contribution_type]
         contribution_data = get_project_locale_contribution_counts(contributions_qs)
 
-        c_count = sum([value["count"] for _, value in contribution_data.items()])
-        p_count = len(contribution_data)
-
-        if c_count == 0:
-            continue
-
-        # Generate localizaton URL and add it to the data dict
+        title = ""
+        for month, data in contribution_data.items():
+            total_count = 0
+            projects = set()
+            for key, info in data.items():
+                total_count += info["count"]
+                # Count number of unique project/locale pairs
+                projects.add(key)
+
+            # Generate title for the localizations belonging to the same contribution type
+            p_count = len(projects)
```

Comment:
This is the logic we use currently, I think it's simpler:

```suggestion
            total_count = sum([info["count"] for _, info in data.items()])
            p_count = len(data)

            # Generate title for the localizations belonging to the same contribution type
```